WinUIGallery/Samples/SamplePages/TitleBarWindow.xaml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Window
+    x:Class="WinUIGallery.Samples.SamplePages.TitleBarWindow"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:WinUIGallery.Samples.SamplePages"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Title="TitleBarWindow">
+    <Window.SystemBackdrop>
+        <MicaBackdrop />
+    </Window.SystemBackdrop>
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="Auto" />
+            <RowDefinition Height="*" />
+        </Grid.RowDefinitions>
+        <TitleBar
+            x:Name="titleBar"
+            Title="WinUI Gallery"
+            BackRequested="TitleBar_BackRequested"
+            IsBackButtonVisible="{x:Bind navFrame.CanGoBack, Mode=OneWay}"
+            IsPaneToggleButtonVisible="True"
+            PaneToggleRequested="TitleBar_PaneToggleRequested"
+            Subtitle="TitleBar sample">
+            <TitleBar.IconSource>
```

Comment:
As @Zakariathr22 pointed out there still is https://github.com/microsoft/microsoft-ui-xaml/issues/10374. Should we use the workaround we used for Gallery in the meantime?